draft-huque-dnsop-ns-revalidation.xml

```diff
@@ -265,6 +274,10 @@
     <section title="Delegation Revalidation" anchor="revalidation">
 
       <t>
+        This documents proposes two mechanism to perform delegation
+	revalidation: an extensive and a simple mechanism. The extensive
+	mechanism:
+
```

Comment:
Okay. Ideally, I'd like to drop the detailed mechanism completely, so we can plan to chat with Paul, and if we all agree, let's try to do that.

Reply:
Agree, I would also like to drop the detailed mechanism.